pwndbg/emu/emulator.py

```diff
@@ -93,6 +165,7 @@ def debug(fmt, args=()) -> None:
     U.UC_ARCH_RISCV: [C.riscv_const.RISCV_INS_ECALL],
 }
 
+# https://github.com/unicorn-engine/unicorn/issues/550
 blacklisted_regs = ["ip", "cs", "ds", "es", "fs", "gs", "ss", "fsbase", "gsbase"]
```

Comment:
FWIW seems like we can emulate `fsbase` and `gsbase` with nowadays unicorn engine with the patch below and don’t need to blacklist the `fsbase` and `gsbase` anymore:
https://github.com/lebr0nli/pwndbg/commit/8b978934b9c9db036c26cf59fd6b6888e7308a24

Before the patch:
<img width="1484" alt="image" src="https://github.com/pwndbg/pwndbg/assets/61896187/2709d8df-59bc-461b-8a64-ae45596ffb8c">
After the patch:
<img width="1479" alt="image" src="https://github.com/pwndbg/pwndbg/assets/61896187/1db83fae-503f-408a-a1bd-8e9dd83c6478">
(But I haven't invested in why the above case shows an empty `()` after the emulated `RDX` value instead of `(0x6161616161616166 - 0x5e47343989b4bd00)` though :p)

----

btw this is probably related to https://github.com/pwndbg/pwndbg/issues/113